docs/source/sms.rst

```diff
@@ -0,0 +1,115 @@
+SMS Data Ingestion
+==================
+Some of the monitors in COSMO rely on data that are found in Science Mission Schedule (SMS) reports.
+These reports are produced outside of the COS Branch and supplied via *txt* files.
+
+A variety of data on exposures is provided in these report files, and in some cases can actually be used to verify
+information in the regular data products themselves.
+
+SMS File Format and Naming Conventions
+--------------------------------------
+The Report files use a human readable table format, which requires special processing to read since it's not a "typical"
+table format (such as *csv*).
+
+The naming convention for these files are comprised of two different ID codes:
+ 1. A "SMS ID" which uniquely identifies a particular SMS report
+ 2. A "Version ID" which identifies the version of a particular SMS
+
+The SMS ID is based on the date that the report was created, and is the first six digits in the report filename.
```

Comment:
replace "is based on the date that the report was created" with "is the start data of the SMS in the form YYDDD"